arm-visualstudio/2014-04-01-preview/swagger/Csm.json

```diff
@@ -0,0 +1,881 @@
+            "schema": {
+              "$ref": "#/definitions/AccountResourceGetResponse"
+            }
+         },
+          "404": {
+            "description": "The Visual Studio Team Services account does not exist."
+         }
+        },
+        "x-ms-examples": {
+            "getAccountResource": {
+                "$ref": "../examples/GetResource.json"
+            }
+        }   	
+      }
+    }
+  },
+  "definitions": {
+    "AccountResourceGetResponse": {
+      "description": "The response to an account resource GET request.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Unique identifier of the resource.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "location": {
+          "description": "Resource location.",
+          "type": "string"
+        },
+        "name": {
+          "description": "Resource name.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "properties": {
+          "description": "Resource properties.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "x-ms-client-flatten": true
+        },
+        "tags": {
+          "description": "Resource tags.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "type": {
+          "description": "Resource type.",
+          "readOnly": true,
+          "type": "string"
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "AccountResourceGroupGetResponse": {
+      "description": "The response to an account resource group GET request.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Array of resource details.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AccountResourceGetResponse"
+          }
+        }
+      }
+    },
+    "AccountResourcePutRequestData": {
+      "description": "The body of a PUT request to modify a Visual Studio account resource.",
+      "type": "object",
+      "properties": {
+        "accountName": {
+          "description": "Gets the account name",
+          "type": "string"
+        },
+        "email": {
+          "description": "Gets the email.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "location": {
+          "description": "Gets or sets the Azure instance location.",
+          "type": "string"
+        },
+        "operationType": {
+          "description": "Gets the type of the operation.",
+          "enum": [
+            "unknown",
+            "create",
+            "update",
+            "link"
+          ],
+          "x-ms-enum": {
+            "name": "Microsoft.VisualStudio.Services.Commerce.AccountResourcePutRequestOperationType"
```

Comment:
seems there are still fully qualified namespace enum in the spec, please correct them